csrankings-v.csv

```diff
@@ -434,6 +434,7 @@ Vladislav Ryzhikov,Birkbeck University of London,https://www.dcs.bbk.ac.uk/about
 Vlado Keselj,Dalhousie University,https://www.cs.dal.ca/~vlado,uDKsmuIAAAAJ
 Vlado Menkovski,TU Eindhoven,https://www.tue.nl/en/university/departments/mathematics-and-computer-science/the-department/staff/detail/ep/e/d/ep-uid/20093057,2s9HUEMAAAAJ
 Vlastislav Dohnal,Masaryk University,https://www.muni.cz/en/people/2952,Vmj26i4AAAAJ
+Vo Nguyen Quoc Bao,PTIT,https://sites.google.com/site/baovnq/home?authuser=0,hOm5kIIAAAAJ

```

Comment:
Not in CS. Please remove or justify how they meet the inclusion criteria as explained in the instructions.

Reply:
Yes, I did remove Mr. Bao.